From bunk@kernel.org  Fri Oct 26 22:19:20 2007
From: Adrian Bunk <bunk@kernel.org>
Date: Sat, 27 Oct 2007 03:06:33 +0200
Subject: PCI: drivers/pci/: remove unused exports
To: Greg KH <gregkh@suse.de>
Cc: linux-pci@atrey.karlin.mff.cuni.cz
Message-ID: <20071027010633.GL30533@stusta.de>
Content-Disposition: inline


This patch removes the following unused exports:
- remove the following unused EXPORT_SYMBOL's:
 - pci-acpi.c: pci_osc_support_set
 - proc.c: pci_proc_detach_bus
- remove the following unused EXPORT_SYMBOL_GPL's:
  - bus.c: pci_walk_bus
  - probe.c: pci_create_bus
  - setup-res.c: pci_claim_resource

Signed-off-by: Adrian Bunk <bunk@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/bus.c       |    1 -
 drivers/pci/pci-acpi.c  |    1 -
 drivers/pci/probe.c     |    1 -
 drivers/pci/proc.c      |    4 ----
 drivers/pci/setup-res.c |    1 -
 5 files changed, 8 deletions(-)

--- a/drivers/pci/bus.c
+++ b/drivers/pci/bus.c
@@ -204,7 +204,6 @@ void pci_walk_bus(struct pci_bus *top, v
 	}
 	up_read(&pci_bus_sem);
 }
-EXPORT_SYMBOL_GPL(pci_walk_bus);
 
 EXPORT_SYMBOL(pci_bus_alloc_resource);
 EXPORT_SYMBOL_GPL(pci_bus_add_device);
--- a/drivers/pci/pci-acpi.c
+++ b/drivers/pci/pci-acpi.c
@@ -188,7 +188,6 @@ acpi_status pci_osc_support_set(u32 flag
 	}
 	return AE_OK;
 }
-EXPORT_SYMBOL(pci_osc_support_set);
 
 /**
  * pci_osc_control_set - commit requested control to Firmware
--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -1140,7 +1140,6 @@ err_out:
 	kfree(b);
 	return NULL;
 }
-EXPORT_SYMBOL_GPL(pci_create_bus);
 
 struct pci_bus *pci_scan_bus_parented(struct device *parent,
 		int bus, struct pci_ops *ops, void *sysdata)
--- a/drivers/pci/proc.c
+++ b/drivers/pci/proc.c
@@ -480,7 +480,3 @@ static int __init pci_proc_init(void)
 
 __initcall(pci_proc_init);
 
-#ifdef CONFIG_HOTPLUG
-EXPORT_SYMBOL(pci_proc_detach_bus);
-#endif
-
--- a/drivers/pci/setup-res.c
+++ b/drivers/pci/setup-res.c
@@ -125,7 +125,6 @@ int pci_claim_resource(struct pci_dev *d
 
 	return err;
 }
-EXPORT_SYMBOL_GPL(pci_claim_resource);
 
 int pci_assign_resource(struct pci_dev *dev, int resno)
 {
